src/main/java/duke/Events.java

```diff
@@ -0,0 +1,28 @@
+package duke;
+
+/**
+ * The Events class which is a subclass of Task,
+ * encapsulates Events objects
+ *
+ * @author  Wee Xin Yang, Markus
+ * @version 0.1
+ * @since   2022-8-24
+ */
+public class Events extends Task {
```

Comment:
Would it be better to name this class `Event` instead of `Events` since it represents a single event and not a group of events?